### DISCUSSION TITLE
expose speaker_a/speaker_b to Home Assistant

### DISCUSSION THREAD

**Discussion by hmoffatt** [vigonotion/aiomusiccast#38] [Ideas]
Would it be possible to expose the speaker_a/speaker_b booleans to Home Assistant? I would like to use them in automations.

Specifically I would like to use them as inputs, rather than setting them. When speakers are off I want to turn on my headphone amp and vice versa.

**Comment by vigonotion** [comment]
This is the library and not directly related to the Home Assistant integration, so this should be an issue in the Home Assistant repository. But good news, there is already an open pull request that implements speaker a/b: https://github.com/home-assistant/core/pull/66925
Sadly, its been sometime and we are waiting for approval

**Reply by hmoffatt** [reply]
Great. Thanks. I would have filed this on the HA core repository but the instructions there say not to open issues for feature requests and I wasn't sure how else to reach out.

**Comment by hmoffatt** [comment]
I see this is in the 2022.5 release. Thanks!
